Makefile

```diff
@@ -41,5 +41,6 @@ circle-run-unit-tests:
 	cd $(DRUPAL_SITE_PATH) && php scripts/run-tests.sh --php /opt/circleci/.phpenv/shims/php --xml $(CIRCLE_TEST_REPORTS)/phpunit \
 	  "Ding! - Ting search unittest"
 	cd $(DRUPAL_SITE_PATH) && php scripts/run-tests.sh --php /opt/circleci/.phpenv/shims/php --xml $(CIRCLE_TEST_REPORTS)/phpunit \
-	  --class ConnieSearchProviderImplementationTestCase
-
+	  --class ConnieSearchSearchProviderImplementationTestCase
+	# Detect failed tests
+	$(DRUPAL_SITE_PATH)/profiles/ding2/tests/detect-test-errors.sh $(CIRCLE_TEST_REPORTS)/phpunit
```

Comment:
Is this *really* necessary? It seems like [an updated Core would ensure that `run-rests.sh` uses the proper exist code](https://www.drupal.org/node/2189345) which in turn will make the build fail on errors.